dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/util/SystemConfig.java

```diff
@@ -90,11 +89,8 @@ public String getVersion(boolean withBuildNumber) {
                 // contains a real build number if this war file was built by
                 // Jenkins)
 
-                try {
-                    buildNumberString = ResourceBundle.getBundle("BuildNumber").getString("build.number");
-                } catch (MissingResourceException ex) {
-                    buildNumberString = null;
-                }
+                buildNumberString = ResourceBundle.getBundle("BuildNumber").getString("build.number");
+                buildNumberString = buildNumberString.isEmpty() ? null : buildNumberString;
```

Comment:
In this case it will not work as we expect it. `BuildNumber` bundle is not provided by default. And it will throw `MissingResourceException`. (note that this code is not using `BundleUtil`)

Reply:
Right.